src/Apps/Artwork/ArtworkApp.tsx

```diff
@@ -75,6 +76,8 @@ const BelowTheFoldArtworkDetails: React.FC<BelowTheFoldArtworkDetailsProps> = ({
 export const ArtworkApp: React.FC<Props> = props => {
   const isNewArtworkSidebarEnabled = useFeatureFlag("fx-force-artwork-sidebar")
   const { artwork, me, referrer, tracking, shouldTrackPageView } = props
+  const showUnlistedArtworkBanner =
+    artwork?.visibilityLevel == "UNLISTED" && artwork?.partner
```

Comment:
wondering if we can remove the partner validation based on the [slack discussion](https://artsy.slack.com/archives/C9ZJYFDNF/p1666805404390339), what do you think?

Reply:
so we get type errors when you try to remove the partner check that I wasn't able to resolve even with doing partner?  in the props of the element. It still works, but typescript doesn't like it. And it would fail in CI

Reply:
[side note]: Wonder if there are stricter type checks in eigen, I actually was getting yelled at by typescript for doing the string check, 
Needed it to be of [type enum ](https://github.com/artsy/eigen/pull/7540/files#diff-9f418fccaa09fb8ea23d492fa300d6e3be69881dc6fbe84fc3c979e114e36533R36-R40) and then [check the object](https://github.com/artsy/eigen/pull/7540/files#diff-9f418fccaa09fb8ea23d492fa300d6e3be69881dc6fbe84fc3c979e114e36533R123)